### PR TITLE
Fix hydroponics seeds never being able to lose some traits through mutations

### PR DESCRIPTION
**Pull request by gurfan**
## What this does
fixes seeds never being able to lose some mutations like vine spreading

## Why it's good
k

## Changelog

:cl:
 * bugfix: Fixed hydroponics seeds never being able to lose some traits through random mutations.

### CODE REVIEW

File: code/modules/hydroponics/hydroponics_mutations.dm [hunk 1]
```diff
@@ -169,8 +169,8 @@
 						seed.fluid_consumption -= change*seed.fluid_consumption
 					generic_mutation_message("rustles!")
 				if(PLANT_VORACIOUS)
-					//clever way of going from 0 to 1 to 2. Flips between 1 and 2.
-					seed.voracious = seed.voracious % 2 + 1
+					//clever way of going from 0 to 1 to 2.
+					seed.voracious = (seed.juicy + 1) % 3
```
Comment (Kurfursten) [inline code/modules/hydroponics/hydroponics_mutations.dm]: wait a minute that var name...

File: code/modules/hydroponics/hydroponics_mutations.dm [hunk 2]
```diff
@@ -200,7 +200,7 @@
 					seed.maturation -= round(min(hardcap - hardcap/2*round(log(10,hardcap/seed.maturation*100),0.01),max_change*seed.maturation),0.1)
 					generic_mutation_message("wriggles!")
 				if(PLANT_SPREAD)
-					seed.spread = seed.spread % 2 + 1
+					seed.spread = (seed.juicy + 1) % 3
```
Comment (Kurfursten) [inline code/modules/hydroponics/hydroponics_mutations.dm]: return of the juicy var...

**Review by Kurfursten — CHANGES_REQUESTED** [review]
Fix it

**Comment by gurfan** [comment]
fixed

**Review by Kurfursten — APPROVED** [review]

**Pull request merged**